src/features/game/types/decorations.ts

```diff
@@ -100,6 +105,74 @@ export const DECORATION_DIMENSIONS: Record<DecorationName, Dimensions> = {
     height: 1,
     width: 1,
   },
+  "Abandoned Bear": {
+    height: 1,
+    width: 1,
+  },
+  "Turtle Bear": {
+    height: 1,
+    width: 1,
+  },
+  "T-Rex Skull": {
+    height: 2,
+    width: 2,
+  },
+  "Sunflower Coin": {
+    height: 2,
+    width: 2,
+  },
+  Foliant: {
+    height: 2,
+    width: 2,
+  },
+  "Skeleton King Staff": {
+    height: 3,
+    width: 2,
+  },
+  "Lifeguard Bear": {
+    height: 1,
+    width: 1,
+  },
+  "Snorkel Bear": {
+    height: 1,
+    width: 1,
+  },
+  "Parasaur Skull": {
+    height: 2,
+    width: 2,
+  },
+  "Golden Bear Head": {
+    height: 2,
+    width: 2,
+  },
+  "Pirate Bear": {
```

Comment:
<img width="260" alt="Screen Shot 2023-02-01 at 3 41 01 pm" src="https://user-images.githubusercontent.com/11745561/215951651-da4abb8b-5e67-4e82-bcac-fe700f077c70.png">

The pirate bear looks a bit to small

Reply:
Same for the bear with a lifeguard (or any SFTs in general with pixel width slightly greater than 16px)